images/Dockerfile-clientserver

```diff
@@ -1,12 +1,16 @@
 FROM quay.io/redhat-user-workloads/rhtas-tenant/rhtas-cli-1-0-beta/cosign@sha256:1a34ac0ee18d144b18a3f18333216e87709018acb6fea42892b5e1a17e2c8af7 AS cosign-image
 FROM quay.io/redhat-user-workloads/rhtas-tenant/rhtas-cli-1-0-beta/gitsign@sha256:a8e8fb362beb5a9a10f11877f676d4d6399b4f66fe9bd15c3bc352748707b98d AS gitsign-image
+FROM quay.io/redhat-user-workloads/rhtas-tenant/rhtas-cli-1-0-beta/rekor-cli@sha256:7da0656cc35c0fce3e62f80d8a1e72eac78a4caf6e636b27e598b98b77d8f9b3 AS rekor-image
 
 FROM registry.redhat.io/rhel8/httpd-24:latest
 
 RUN mkdir -p /var/www/html/clients
 
 COPY --from=cosign-image /usr/local/bin/cosign /var/www/html/clients/
 COPY --from=gitsign-image /usr/local/bin/gitsign /var/www/html/clients/
+COPY --from=rekor-image /usr/local/bin/rekor_cli_darwin_amd64 /var/www/html/clients/
+COPY --from=rekor-image /usr/local/bin/rekor_cli_linux_amd64 /var/www/html/clients/
+COPY --from=rekor-image /usr/local/bin/rekor_cli_windows_amd64.exe /var/www/html/clients/
 
```

Comment:
Now I think it will be better if we create 
```
/var/www/html/clients/darwin
/var/www/html/clients/windows
/var/www/html/clients/linux
```
and do like 
```
COPY --from=gitsign-image /usr/local/bin/rekor_cli_linux_amd64 /var/www/html/clients/linux/rekor-cli
COPY --from=cosign-image /usr/local/bin/cosign_cli_linux_amd64 /var/www/html/clients/linux/cosign
COPY --from=gitsign-image /usr/local/bin/gitsign_cli_linux_amd64 /var/www/html/clients/linux/gitsign
(the cosign, gitsign are not named like that yet but will be)

then the same for /var/www/html/clients/darwin, windows
```
because then they will be named `cosign, gitsign, rekor-cli` in the consoleCLIDownload - and will save the user from downloading then having to `mv rekor_cli_linux_amd64 rekor-cli` 
make sense?